types/react-native-auth0/index.d.ts

```diff
@@ -0,0 +1,171 @@
+// Type definitions for react-native 1.2
+// Project: https://github.com/auth0/react-native-auth0
+// Definitions by: Andrea Ascari <https://github.com/ascariandrea>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.6
+
+/**
+ * Auth
+ */
+
+export interface AuthorizationUrlParams {
+    responseType: string;
+    redirectUri: string;
+    state: string;
+}
+
+export interface CreateUserParams<T extends any> {
```

Comment:
This is an unnecessary constraint.

Reply:
Right, but why `extends any`? It's not adding anything here.

Reply:
Removed!